package.json

```diff
@@ -67,6 +70,7 @@
     "siwe": "^1.1.6",
     "swagger-ui-express": "^4.3.0",
     "tsoa": "^5.1.1",
+    "tweetnacl": "^1.0.3",
```

Comment:
what is this lib for?

Reply:
I just wonder whether it would be better to move it to devDependencies